### PR TITLE
Add other storage system

### PR DESCRIPTION
**Pull request by wnxn**
Signed-off-by: Xin Wang <wileywang@yunify.com>

### CODE REVIEW

File: sig-storage/concepts-and-designs/csi-plugin.md [hunk 1]
```diff
@@ -16,3 +16,7 @@ For QingCloud IaaS, we have developed [QingCloud CSI](https://github.com/yunify/
 ### QingStor Storage System
 
 We have developed [QingStor CSI](https://github.com/yunify/qingstor-csi) plugin to integrate Kubernetes with QingStor NeonSAN storage system. If users want to use Kubernetes on bare-metal, they may need to store data in QingStor storage system. The QingStor CSI plugin will also implement the latest CSI features.
+
+### Others Storage System
+
+Because KubeSphere based on Kubernetes, KubeSphere can utilze various storage systems through thier CSI plugin which supports [dynamic volume provisioning](https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/). You can find CSI plugins in [Kubernetes CSI drivers list](https://kubernetes-csi.github.io/docs/drivers.html).
```
Comment (rayzhou2017) [inline sig-storage/concepts-and-designs/csi-plugin.md:22]: typo: their --> their

Comment (rayzhou2017) [inline sig-storage/concepts-and-designs/csi-plugin.md:22]: Because KubeSphere based on Kubernetes --> Because KubeSphere is based on Kubernetes

**Comment by wnxn** [comment]
@rayzhou2017 Please take a look.

**Pull request merged**